hack/verify-codegen.sh

```diff
@@ -19,7 +19,7 @@ set -o nounset
 set -o pipefail
 
 readonly REPO_ROOT_DIR="$(git rev-parse --show-toplevel)"
-readonly TMP_DIFFROOT="$(mktemp -d -p ${REPO_ROOT_DIR})"
+readonly TMP_DIFFROOT="$(TMPDIR=${REPO_ROOT_DIR} mktemp -d)"
```

Comment:
Looking at the man pages `TMPDIR` (for mac) is only used if `-t` is set. So setting it in this context doesn't do anything.

I think we can just drop it and use `$(mktemp -d)`

Reply:
Okay, done. Thank you.